v2/mfa_recovery_codes.go

```diff
@@ -0,0 +1,18 @@
+package omise
+
+// MfaRecoveryCodes represents resource object.
+type MfaRecoveryCodes struct {
```

Comment:
acronym in go should remains all capital case, `Mfa` -> `MFA` if this is Multi-Factor Authentication here